pkg/kmetrics/record.go

```diff
@@ -46,76 +47,85 @@ func RecordKustomizeFieldCountData(ctx context.Context, fieldCountData *Kustomiz
 	recordKustomizeTopTierMetrics(ctx, fieldCountData.TopTierCount)
 }
 
+func record(ctx context.Context, ms ...stats.Measurement) {
+	stats.Record(ctx, ms...)
+	if klog.V(5).Enabled() {
+		for _, m := range ms {
+			klog.V(5).Infof("Metric recorded: { \"Name\": %q, \"Value\": %#v, \"Tags\": %s }", m.Measure().Name(), m.Value(), tag.FromContext(ctx))
```

Comment:
`.V(5)` is not needed as `klog.V(5)` is already enabled in this case.

Reply:
The reason I use the levels log inside the if enabled statement is because it makes it easier to programatically refactor later without requiring manual changes. I remember an email thread a few years back about how K8s was hard to refactor in these cases when they decided what the levels should mean. But you're right that it's redundant. It's only redundant when the log level is high tho, so it's not a big tax. Do you still want me to simplify it?

Reply:
Fixed, for consistency with other -v=5 logs.